src/libraries/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -64,9 +64,15 @@ public static Uri[] GetEchoServerList()
                 if (PlatformDetection.IsFirefox)
                 {
                     // https://github.com/dotnet/runtime/issues/101115
-                    return [RemoteEchoServer];
+                    // [ActiveIssue("https://github.com/dotnet/runtime/issues/110578)]
+                    // return [RemoteEchoServer];
```

Comment:
```suggestion
                    // return [RemoteEchoServer];
                    return [];
```
Based on the issue linked above